docker/scripts/install_scpca_deps.sh

```diff
@@ -30,51 +30,51 @@ apt-get -y --no-install-recommends install \
 
 #### R packages
 ###############
-
-# Set CRAN mirror to allow access to more recent matrixStats
-CRAN="https://packagemanager.rstudio.com/cran/__linux__/focal/2021-08-27"
-echo "options(repos = c(CRAN = '${CRAN}'), download.file.method = 'libcurl')" >> ${R_HOME}/etc/Rprofile.site
-# update all to that repo
-Rscript -e "update.packages(ask = FALSE)"
-
-# this comes first since bioconductor packages are
-# dependent on updated matrixStats
-install2.r --error --skipinstalled -n $NCPUS \
-    BiocManager \
-    DBI \
-    devtools \
-    here \
-    kableExtra \
-    matrixStats \
-    optparse \
-    rmarkdown \
-    rprojroot \
-    RSQLite \
-    tidyverse
-
-
-##########################
-# Install bioconductor packages
-Rscript -e "withCallingHandlers(
-  BiocManager::install(c( \
-    'AnnotationHub', \
-    'Biostrings', \
-    'bluster', \
-    'BSgenome', \
-    'DropletUtils', \
-    'eisaR', \
-    'ensembldb', \
-    'fishpond', \
-    'GenomicFeatures', \
-    'miQC', \
-    'scater', \
-    'scran', \
-    'scuttle', \
-    'SingleCellExperiment', \
-    'SummarizedExperiment', \
-    'tximport'), \
-    update = FALSE), \
-  warning = function(w) stop(w))"
-
-rm -rf /tmp/downloaded_packages
-rm -rf /tmp/Rtmp*
+#
+# # Set CRAN mirror to allow access to more recent matrixStats
+# CRAN="https://packagemanager.rstudio.com/cran/__linux__/focal/2021-08-27"
+# echo "options(repos = c(CRAN = '${CRAN}'), download.file.method = 'libcurl')" >> ${R_HOME}/etc/Rprofile.site
+# # update all to that repo
+# Rscript -e "update.packages(ask = FALSE)"
+#
+# # this comes first since bioconductor packages are
+# # dependent on updated matrixStats
+# install2.r --error --skipinstalled -n $NCPUS \
+#     BiocManager \
+#     DBI \
+#     devtools \
+#     here \
+#     kableExtra \
+#     matrixStats \
+#     optparse \
+#     rmarkdown \
+#     rprojroot \
+#     RSQLite \
+#     tidyverse
+#
+#
+# ##########################
+# # Install bioconductor packages
+# Rscript -e "withCallingHandlers(
+#   BiocManager::install(c( \
+#     'AnnotationHub', \
+#     'Biostrings', \
+#     'bluster', \
+#     'BSgenome', \
+#     'DropletUtils', \
+#     'eisaR', \
+#     'ensembldb', \
+#     'fishpond', \
+#     'GenomicFeatures', \
+#     'miQC', \
+#     'scater', \
+#     'scran', \
+#     'scuttle', \
+#     'SingleCellExperiment', \
+#     'SummarizedExperiment', \
+#     'tximport'), \
+#     update = FALSE), \
+#   warning = function(w) stop(w))"
+#
+# rm -rf /tmp/downloaded_packages
+# rm -rf /tmp/Rtmp*
```

Comment:
I think we can go ahead and remove the commented out portion now? Unless there was a reason you were keeping it in there?